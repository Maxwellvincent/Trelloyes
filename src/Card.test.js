import React from 'react';
import ReactDOM from 'react-dom';
import Card from './Card';
import renderer from 'react-test-renderer';
import { render } from '@testing-library/react';

describe('This is a test for smoke and snapshot on the card component', ()=> {
    it ('this is a smoke test to see if the component renders', () => {
        const div = document.createElement('div');
        ReactDOM.render(<Card />,div);
        ReactDOM.unmountComponentAtNode(div);
    });

    it('this is to take a snapshot of the UI and make sure it renders correctly', ()=> {
        const tree = renderer.create(<Card className="card"/>).toJSON();
        expect(tree).toMatchSnapshot();
    })
})