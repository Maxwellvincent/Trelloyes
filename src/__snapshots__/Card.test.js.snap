// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`This is a test for smoke and snapshot on the card component this is to take a snapshot of the UI and make sure it renders correctly 1`] = `
<div
  className="Card"
>
  <h3 />
  <p />
</div>
`;
